v2/azure-arm.yaml

```diff
@@ -2373,6 +2385,12 @@ objectModelConfiguration:
       ManagedClusters_MaintenanceConfiguration:
         $exportAs: MaintenanceConfiguration
         $supportedFrom: v2.11.0
+      ScaleProfile:
```

Comment:
There's a newer version of ManagedCluster in `2025-08-01`, does this not go there?

Reply:
No - the preview version (I think it was `2025-07-02preview` ?) got inserted in between the two.